x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/utils/indices.ts

```diff
@@ -104,9 +104,15 @@ export function getLastUpdated(index?: ElasticsearchIndexWithIngestion): string
   return isConnectorIndex(index) ? index.connector.last_synced ?? 'never' : null;
 }
 
-export function indexToViewIndex(index: ConnectorIndex): ConnectorViewIndex;
-export function indexToViewIndex(index: CrawlerIndex): CrawlerViewIndex;
-export function indexToViewIndex(index: ElasticsearchIndexWithIngestion): ApiViewIndex {
+export function indexToViewIndex(
```

Comment:
question: I'm a bit confused by this change. What is this accomplishing?

Reply:
TS was complaining about this, as I was passing `ElasticsearchIndexWithIngestion` typed index, where it was not compatible with first two overloads. 

After talking with @TattdCodeMonkey offline, I think better solution is to change index to `ElasticsearchIndex` rather.